roles/unpriviledged_user/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+unpriviledged_user_username: vagrant
+# unpriviledged_user_import_ssh_pub_key: 
```

Comment:
Given this is commented out, can it be dropped?

Reply:
I define all variables which are use by script (my habit.).  It is easier for next using. I can see all possible options (especialy correct names or some comments for it) on the one place and I don't have to read whole source code. In this case, the variable unpriviledged_user_import_ssh_pub_key undefined in default.
If you have a problem with this, I can remove this.

Reply:
I'd prefer it removed or set to a default value. The point of defaults is just that, to set variables and give them a default in case no user supplies them.